### PR TITLE
[15.0][ADD] survey_placeholder: New module

### PR DESCRIPTION
**Pull request by chienandalu**
Basically a backport of https://github.com/odoo/odoo/pull/78682 from v16

cc @Tecnativa TT40984

### CODE REVIEW

File: survey_placeholder/readme/ROADMAP.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+In v16 the feature is already included in core.
```
Comment (pedrobaeza) [inline survey_placeholder/readme/ROADMAP.rst]:
```suggestion
* In v16, the feature is already included in core.
```

**Review by pedrobaeza — APPROVED** [review]

File: survey_placeholder/readme/ROADMAP.rst (hunk 1 above)
Comment (chienandalu) [reply survey_placeholder/readme/ROADMAP.rst]: Done

File: survey_placeholder/views/survey_question_views.xml [hunk 2]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="survey_question_form" model="ir.ui.view">
+        <field name="model">survey.question</field>
+        <field name="inherit_id" ref="survey.survey_question_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//div[hasclass('oe_title')]" position="after">
```
Comment (stefan-tecnativa) [inline survey_placeholder/views/survey_question_views.xml:7]: In 15 this class is applied in the title section only, so the field is being placed under the questions name, not inside the options tab: [](https://github.com/odoo/odoo/blob/15.0/addons/survey/views/survey_question_views.xml#L16C64-L16C64)

Comment (chienandalu) [reply survey_placeholder/views/survey_question_views.xml:7]: Yes, the options in the question form are different in v16. So I decided to put it close to the question title which is quite related to the placeholder

Comment (stefan-tecnativa) [reply survey_placeholder/views/survey_question_views.xml:7]: Ah, undestood. It looks correct there for me.

**Review by stefan-tecnativa — APPROVED** [review]

**Pull request merged**